### PR TITLE
Fixed broken dependencies

### PR DESCRIPTION
**Pull request by redfave**
Dependencies sind aktualisiert, dass sich sipa builden und ausführen lässt, allerdings laufen die Tests nicht, da der Testrunner nicht mit der neuen Flask Version kompatibel ist.

### CODE REVIEW

**Comment by lukasjuhrich** [comment]
Hey, thanks for the updates! Hope it's okay if I continue in English.

Thanks for your contribution! Dear god, that setup is _old_… `flask-script` received its last update in 2017, and we don't even need the 

Most of the complexity in the setup which breaks now isn't really needed. I think I can safely tear it down.

To give some context:
Historically, sipa was a unified frontend for separate data backends (Wundtstraße, Hochschulstraße, Zeunerstraße, to name a few) – and thus we needed to set up a lot of fake backends, like a postgresql database, an LDAP, a mysql database, etc.pp.
Furthermore, we historically used `unittest`, but many of us switched to `nosetests`, and we supported both test runners ever since. That's completely unnecessary nowadays and we want to use `pytest` anyway due to its simplicity.

For the first reason, we created a `flask_script` wrapper CLI to essentially just wait for all the backends (database servers) to be ready before starting the test.

In the present, we just have one HTTP API, (which we haven't written any tests for, appearently).
But we won't need something as complicated like a separate docker container.

tl;dr: essentially, #431. I'll fix what's necessary for this PR to pass through.

File: build/requirements/requirements.txt [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
+Flask-WTF~=0.15.1
 PyMySQL~=0.7.11
 blinker~=1.4.0
 factory-boy~=2.9.2
 psycopg2~=2.8.4
 Flask-QRcode~=2.0.2
-typing-extensions~=3.7.2
 email_validator~=1.1.1
-Werkzeug~=1.0.0
+Werkzeug==2.0.3
```
Comment (lukasjuhrich) [inline build/requirements/requirements.txt:19]: Please use a [compatible release clause](https://peps.python.org/pep-0440/#version-specifiers) here:
```suggestion
Werkzeug~=2.0.3
```

**Comment by redfave** [comment]
> Please use a [compatible release clause](https://peps.python.org/pep-0440/#version-specifiers) here:

I used specific version matching because this is the latest release which doesn't break the code base.
Newer releases removed the `safe_str_cmp` method, which is used in sipa. I think an update of Werkzeug to the latest version available should be an issue of its own.


> @redfave: To finish this PR off, could you `git pull --rebase`, and
> 
>     * [ ]  take a look at the other outdated packages and
> 
>     * [ ]  maybe even upgrade to python 3.10 ([Upgrade to Python3.10 #437](https://github.com/agdsn/sipa/issues/437))?

I would like to help, but I'm not an active member (yet) and saw the sipa code base yesterday at the code evening for the first time.
As I don't know the architecture of the application and have only rudimentary python skills, I think the risk of getting stuck at some point is too high also because I can't reach out to you guys directly as I'm not an active member.
I could offer to code together on Sunday evening either online or in person if you like.

Kind regards

**Comment by lukasjuhrich** [comment]
> I could offer to code together on Sunday evening either online or in person if you like.

Unfortunately I was busy upgrading the postgres cluster today. We'll find another date (C:

File: build/requirements/requirements.txt (hunk 1 above)
Comment (lukasjuhrich) [reply build/requirements/requirements.txt]: Fixed in 34d26198eae2f27083038247e9fea5b6a413ba0

**Comment by lukasjuhrich** [comment]
@redfave sorry for merging this so late, but the failing loginmanager tests were quite tricky :-)
